learning/tour-of-beam/frontend/lib/pages/tour/state.dart

```diff
@@ -67,6 +67,26 @@ class TourNotifier extends ChangeNotifier with PageStateMixin<void> {
 
   String? get currentUnitId => currentUnitController?.unitId;
   UnitContentModel? get currentUnitContent => _currentUnitContent;
+  bool get doesCurrentUnitHaveSolution =>
+      currentUnitContent?.solutionSnippetId != null;
+  bool _showSolution = false;
+  bool get showSolution => _showSolution;
```

Comment:
`isShowingSolution` is less ambiguous.
That will imply `toggleShowingSolution()`